tests/sentry/sentry_metrics/test_postgres_indexer.py

```diff
@@ -38,6 +39,41 @@ def test_indexer(self):
         assert PGStringIndexer().reverse_resolve(1234) is None
 
 
+class StaticStringsIndexerTest(TestCase):
+    def setUp(self) -> None:
+        self.indexer = StaticStringsIndexerDecorator()
+        StringIndexer.objects.create(organization_id=0, string="release", id=13)
+        StringIndexer.objects.create(organization_id=0, string="production", id=12)
+        StringIndexer.objects.create(organization_id=0, string="environment", id=7)
```

Comment:
Does the test require these entries in the database? `bulk_record` gets them from the static dict, right?

Reply:
ah yeah good point, had copied this over from before but needs to be updated